docs/detections/prebuilt-rules/rule-details/bypass-uac-via-sdclt.asciidoc

```diff
@@ -1,66 +0,0 @@
-[[bypass-uac-via-sdclt]]
-=== Bypass UAC via Sdclt
```

Comment:
Was this rule deprecated or something?

Reply:
yep. Looks like previously removed rules were handled like this:
![image](https://user-images.githubusercontent.com/16747370/112267593-8d2d3f80-8c2a-11eb-854c-aef466c1d007.png)

Thought I think `deprecated` vs `removed` is better verbiage. Also, how deprecated rules are handled in Kibana is in flux and may change.

I will see how to incorporate this to these docs (even if just manually)

Reply:
see the edited note in the description